src/overlays/actors/ovl_En_Maruta/z_en_maruta.c

```diff
@@ -57,7 +57,7 @@ u8 D_80B386C0[] = {
     0xFF, 0x2B, 0xD4, 0x17, 0xE8, 0x55, 0xAA, 0x0F, 0xF0,
 };
 
-s32 D_80B386CC[] = {
+s32 D_80B386CC[PLAYER_MWA_MAX] = {
```

Comment:
Again, I don't really see that adding the explicit size helps here... much better to document what it actually is.

Reply:
My main reason for doing this is if somebody for some reason adds an extra value to this enum, then the array would grow accordingly by adding extra zeroes, so this would not produce an OoB access in case that person forgets to add the extra value here.
Also, I think this way is easier to find the arrays that need to be adjusted when an extra value is added to the enum

Reply:
I think actual labels on the elements, or use of the enum where appropriate, are likely a better way to solve this than declaring the dimension, since you can't tell it's wrong at compile time anyway.

Reply:
I'd probably agree with Elliptic here. TBH, I would probably prefer the OOB reads as I feel I'd more likely see that, rather than a silent 0 that while doesn't cause obvious problems, might still give incorrect results and is harder to see.